### DISCUSSION TITLE
PIMCORE-X

### DISCUSSION THREAD

**Discussion by ismetsener** [pimcore/pimcore#8666] [General]
Hi,

I read on the road map about the release date of Pimcore-X  [https://pimcore.com/en/developers/roadmap]
The release date for the Open Source Community Edition is March 31, 2021

But it still not there: https://github.com/pimcore/pimcore
when can we expect it?

with kind regards,
Ismet

**Comment by NiklasBr** [comment]
It changed to April 30 not long ago: https://github.com/pimcore/pimcore/milestone/70

**Reply by ismetsener** [reply]
OK, thanks
I am looking forward to download it :)

**Reply by dvesh3** [reply]
If you would like to try beta version version https://github.com/pimcore/skeleton#getting-started-pimcore-x-beta

**Reply by nirbanc2020** [reply]
Anyone installed pimcore X beta1 or beta2 in local machine?I am unable to install it.

**Reply by nirbanc2020** [reply]
> If you would like to try beta version version https://github.com/pimcore/skeleton#getting-started-pimcore-x-beta

Have you installed pimcore X beta1 or beta2 in local machine?I am unable to install it.

**Reply by dvesh3** [reply]
@nirbanc2020 what's the issue?

**Reply by nirbanc2020** [reply]
I have used this command for installing pimcore X
COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/skeleton:10.0.x-dev my-project
cd ./my-project
./vendor/bin/pimcore-install

but after creating virtual hosts the message showing in browser is handle($request); $response->send(); $kernel->terminate($request, $response);

**Reply by nirbanc2020** [reply]
My php version is 8.0.3.Is it right?

**Reply by dvesh3** [reply]
ok, that's not Pimcore issue. please check system requirements here: https://pimcore.com/docs/pimcore/master/Development_Documentation/Installation_and_Upgrade/System_Requirements.html

**Reply by nirbanc2020** [reply]
It is due to mpm_event enable.  run sudo a2dismod mpm_event.Now it is working..  In bottom of admin panel, it is showing the version Pimcore Platform (dev-master|71bd54abaebc8f8b94c57b6445fd14eb23e8aa7d)  Is it pimcore X?

**Reply by dvesh3** [reply]
yes, that's Pimcore X but not the beta version. I fixed the composer.json https://github.com/pimcore/demo/commit/b406c74fababae685521414be7f0f583fa82b9cb so it should download Pimcore X beta. thanks!

**Reply by nirbanc2020** [reply]
what should I have to do? Should I change the line from "pimcore/pimcore": "10.0.*@dev" to "pimcore/pimcore": "10.0.*@beta" and run composer update?

**Comment by ismetsener** [comment]
I installed the pimocreX demo version (php8+mysql8)
COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/demo:10.0.x-dev my-project --ignore-platform-req=php
with mysql-8  I get an error 
with mysql-5.7  no error

but when yoiu install pimcore-X (without demo) it works withmysql-8

one problem is when you try to update or install for example core-shop or  worklow-gui 
you will get an error:
unzerdev/php-sdk 1.1.1.1 requires php ~7.1.0|~7.2.0|~7.3.0|~7.4.0 -> your php version (8.0.3) does not satisfy that requiremen

so some compenents work on php7 and not on php8

**Comment by ismetsener** [comment]
Problem 1
    - pimcore/payment-provider-unzer 1.0.x-dev is an alias of pimcore/payment-provider-unzer dev-master and thus requires it to be installed too.
    - pimcore/payment-provider-unzer[dev-master, v1.0.0, ..., v1.0.1] require unzerdev/php-sdk ^1.1.0.0 -> satisfiable by unzerdev/php-sdk[1.1.0.0, 1.1.1.0, 1.1.1.1].
    - unzerdev/php-sdk[1.1.0.0, ..., 1.1.1.1] require php ~7.1.0|~7.2.0|~7.3.0|~7.4.0 -> your php version (8.0.3) does not satisfy that requirement.
    - Root composer.json requires pimcore/payment-provider-unzer ^1.0 -> satisfiable by pimcore/payment-provider-unzer[v1.0.0, v1.0.1, 1.0.x-dev (alias of dev-master)].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
[root@pimcore pimcore-10]#

**Reply by dvesh3** [reply]
yes, this is known issue. please see https://github.com/unzerdev/php-sdk/pull/22

**Reply by ismetsener** [reply]
Pimcore-10  works with  php8
but some extensions like  unzerdev/php-sdk  works only with php7
can we install also php7 (multi php versions same machine)

**Reply by dvesh3** [reply]
pimcore/payment-provider-unzer 1.0.3 now supports php 8 (with forked Unzer SDK repo for the time being)

**Comment by ismetsener** [comment]
what about  add-ons on market-place
are they also working on there plugins so that it works in PIMCORE-X?

**Reply by dvesh3** [reply]
@ismetsener you have to check for individual bundles on Github.
